src/gui/editors/AutomationEditor.cpp

```diff
@@ -332,34 +330,14 @@ void AutomationEditor::drawLine( int x0In, float y0, int x1In, float y1 )
 
 	float yscale = deltay / ( deltax );
 
-	if( x0 < x1 )
-	{
-		xstep = AutomationClip::quantization();
-	}
-	else
-	{
-		xstep = -( AutomationClip::quantization() );
-	}
-
-	float lineAdjust;
-	if( y0 < y1 )
-	{
-		ystep = 1;
-		lineAdjust = yscale;
-	}
-	else
-	{
-		ystep = -1;
-		lineAdjust = -( yscale );
-	}
+	int xstep = (x0 < x1 ? 1 : -1) * AutomationClip::quantization();
+	int ystep = y0 < y1 ? 1 : -1;
+	float lineAdjust = (y0 < y1 ? 1 : -1) * yscale;
```

Comment:
Is this just `ystep` now?
```suggestion
	float lineAdjust = ystep * yscale;
```